Add `The Followpoint: Spectator, the osu!catch Mapping Pioneer` news post (#13175)
ppy/osu-wiki: news/2025/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer.md

---
layout: post
title: "The Followpoint: Spectator, the osu!catch Mapping Pioneer"
date: 2025-04-30 09:00:00 +0000
series: The Followpoint
---

> Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too little mapper interviews"? Well, today is your lucky day, as we are interviewing Spectator, one of the oldest osu!catch mappers out there!

Reviewer: ```suggestion
Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too few mapper interviews?" Well, today is your lucky day, as we are interviewing Spectator, one of the oldest osu!catch mappers out there!
```

![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/banner.jpg)

> You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many works of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories. He's also a Beatmap Nominator, and not just a regular one, but an [Elite Nominator](/wiki/People/Elite_Nominators) with 145 beatmaps pushed to Ranked in 2024!

Reviewer 2: a GMT as well

Author: Not sure how well it will work in the current intro but alrighty

> You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many maps and difficulties of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories, and of course a Beatmap Nominator, and not a regular one, he's an [Elite Nominator](/wiki/People/Elite_Nominators), personally nominating 145 beatmaps in 2024!

Reviewer: ```suggestion
You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many works of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories. He's also a Beatmap Nominator, and not just a regular one, but an [Elite Nominator](/wiki/People/Elite_Nominators) with 145 beatmaps pushed to Ranked in 2024!
```

> So now that you have an idea how big of community figure Spectator is for osu!catch — let's get to the interview [me](https://osu.ppy.sh/users/18152711) and [-Isla-](https://osu.ppy.sh/users/17745759) conducted with him!

Reviewer: credit is both at the top and bottom of the post, and we very rarely refer to the authors in first person

```suggestion
Now that you have an idea how big of a community figure Spectator is for osu!catch, let's get to hear what insights he has to bring! This time, [-Isla-](https://osu.ppy.sh/users/17745759) is joining in with questions as well.
```

Author: Alright, gotcha! I wasn't sure how I should write this, so thank you for clarifying.

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Hi Spectator! Let's begin with the usual, how would you introduce yourself to the people who don't know you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

Hello everyone, I'm Spectator. I've been playing osu!, mainly in the osu!catch mode for almost 15 years. I used to focus on playing, but these days I'm probably more known as a mapper. I also nominate maps as a Beatmap Nominator and help moderate the osu!catch community as a [Global Moderator](/wiki/People/Global_Moderation_Team).

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/17745759" style="background-image: url('/wiki/shared/avatars/-Isla-.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/17745759">-Isla-</a></p>

> How did you find osu! and what was your impression of it?

Reviewer 2: ```suggestion
How did you find osu!, and what was your impression of it?
```

Why did you choose to be an osu!catch player among all game modes there are?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

I played standard too, but I was just better at osu!catch, which made it more fun and motivating to grind.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Is there a specific memory from osu! that stands out to you? If so, which one?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

> There are a lot of great memories, but the most memorable one for me was the Grand Finals match of the [osu! Catch The Beat World Cup 2014](/wiki/Tournaments/CWC/2014). Our opponent was Germany, a team full of outstanding players like [NoteKuroi](https://osu.ppy.sh/users/186642), [Nyan-Zapo](https://osu.ppy.sh/users/480676), and [DeathXHunter](https://osu.ppy.sh/users/405326). Due to a DDoS attack, we had to replay the match without a stream, and it was one of the most nerve-wracking moments of my osu! tournament career. But somehow, I performed better in the rematch and managed to win the World Cup for the first time. Even to this day, those three players from Germany remain some of my best rivals in the game, and I'll always be grateful to them for the unforgettable memories.

Reviewer 2: ```suggestion
There are a lot of great memories, but the most memorable one for me was the Grand Finals match of the [osu! Catch The Beat World Cup 2014](/wiki/Tournaments/CWC/2014). Our opponent was Germany, a team full of outstanding players like [NoteKuroi](https://osu.ppy.sh/users/186642), [Nyan-Zapo](https://osu.ppy.sh/users/480676), and [DeathXHunter](https://osu.ppy.sh/users/405326).

Due to a DDoS attack, we had to replay the match without a stream, and it was one of the most nerve-wracking moments of my osu! tournament career. But somehow, I performed better in the rematch and managed to win the World Cup for the first time. Even to this day, those three players from Germany remain some of my best rivals in the game, and I'll always be grateful to them for the unforgettable memories.
```

---

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/17745759" style="background-image: url('/wiki/shared/avatars/-Isla-.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/17745759">-Isla-</a></p>

What caused you to start mapping in osu!catch? Was there any particular reason or a person that motivated you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

> When I first started mapping, osu!catch specific maps used to be unrankable. I mostly tried mapping just for fun, and received some positive feedback from other players. There was an old forum for every osu!catch players and I used to share my very first maps [there](https://osu.ppy.sh/community/forums/topics/21061?n=3561).

Reviewer: typo

```suggestion
When I first started mapping, osu!catch specific maps used to be unrankable. I mostly tried mapping just for fun, and received some positive feedback from other players. There was an old forum for every osu!catch player and I used to share my very first maps [there](https://osu.ppy.sh/community/forums/topics/21061?n=3561).
```

![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/forum_post.jpg)

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/17745759" style="background-image: url('/wiki/shared/avatars/-Isla-.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/17745759">-Isla-</a></p>

> As someone who's been mapping for a very long time, do you have any tips or advices on how would somebody comfortably and efficiently get into mapping without any previous experience or knowledge of how to map?

Reviewer: grammar
could be trimmed

```suggestion
As someone who's been mapping for a very long time, do you have any tips or advice on how somebody would comfortably and efficiently get into mapping without any previous experience or knowledge?
```

Before moving to the other topic we'd also like to congratulate you on recently achieving Elite Nominator title! Would you mind telling us what was your motivation to contribute so much to the beatmapping scene as a nominator during 2024?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

What brought me back to being a Beatmap Nominator was wanting to see more variety in the ranked section. Back in 2023, there also weren't many harder maps, especially those over 8 stars, so I wanted to help bring more of them into the ranked section.

---

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> Both as a top player and as a mapper, what are your thoughts on adding CSR to osu!catch? Would that be a bad or a good thing, and why?

Reviewer: this is a rather new term in the community. shouldn't assume prior knowledge

```suggestion
Both as a top player and as a mapper, what are your thoughts on implementing osu!'s performance point [combo scaling removal](https://osu.ppy.sh/home/news/2024-10-28-performance-points-star-rating-updates#combo-scaling-removal) ("CSR") to osu!catch? Would that be a bad or a good thing, and why?
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

I've mostly shifted to mapping, not just because I don't have enough time to grind, but also because I feel like I no longer stand out as a player.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

You have one specific score, [disinteGration by RINYA.G \[Collapse of the World Machine\] +HDHR](https://osu.ppy.sh/scores/1956830227), pinned in your profile. Is there any story behind this or some other reason it's pinned?

![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/score.jpg)

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

> I was in a Discord voice chat, randomly trying maps with mods, and somehow FC'd it first try. I thought it's pretty unique to not have my top pp score as illness lilin, so I decided to pin it.

Reviewer: what is "illness lilin"? please link

Author: illness lilin is a very famous (from my memory farm) [map in osu!catch](https://osu.ppy.sh/beatmapsets/777109#fruits). I think I forgot to link it, so will do that after applying web changes.

---
layout: post
title: "The Followpoint: Spectator, the osu!catch Mapping Pioneer"
date: 2025-04-30 09:00:00 +0000
series: The Followpoint
---

> Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too few mapper interviews?" Well, today is your lucky day, as we are interviewing Spectator, one of the oldest osu!catch mappers out there!

Reviewer 3: ```suggestion
Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too few mapper interviews?" Well, today is your lucky day, as we are interviewing Spectator, one of the most iconic osu!catch mappers out there!
```

![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/banner.jpg)

> You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many works of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories. He's also a Beatmap Nominator, and not just a regular one, but an [Elite Nominator](/wiki/People/Elite_Nominators) with 145 beatmaps pushed to Ranked in 2024! And in addition to all this, Spectator is also a part of the [Global Moderation Team](/wiki/People/Global_Moderation_Team)!

Reviewer 3: ```suggestion
You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many works of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories. He's also a Beatmap Nominator, and not just a regular one, but an [Elite Nominator](/wiki/People/Elite_Nominators) with 145 beatmaps pushed to Ranked in 2024! And in addition to all this, Spectator is also a part of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) — truly a jack of all trades!
```

---

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/17745759" style="background-image: url('/wiki/shared/avatars/-Isla-.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/17745759">-Isla-</a></p>

Okay, now let's move on to mapping. What caused you to start mapping in osu!catch? Was there any particular reason or a person that motivated you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>

When I first started mapping, osu!catch specific maps used to be unrankable. I mostly tried mapping just for fun, and received some positive feedback from other players. There was an old forum for every osu!catch player and I used to share my very first maps [there](https://osu.ppy.sh/community/forums/topics/21061?n=3561).

> ![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/forum_post.jpg)

Reviewer 2: would it be too much to ask to re-take this screenshot without the lines at the top or bottom? the screenshot would fit in better without them

```suggestion
![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/forum_post.jpg)
```

Author: It's fine, will do xD

Author: Should be done
